README.md

```diff
@@ -1,2 +1,37 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
 # Leaderboard
 The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.
+
+![screenshot](./photo.png)
+
+## Built With
+
+- Javascript
+- Webpack
+- HTML 5
+- CSS 3
+
```

Comment:
- [ ] The readme file is missing information on how a user like me can get this project and run it on my local machine.